site/docs/assets/css/extra.css

```
@@ -44,6 +44,77 @@
+}
+.lk-subscribe .lk-subscribe-fine a {
+  color: inherit;
+}
+.lk-visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0); /* fallback for older browsers */
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)